src/misc/image.v1.yaml

```diff
@@ -8,7 +8,7 @@ allOf:
             title: IIIF Image
             type: string
             format: uri
-            pattern: ^https://[^/]+(/[^?#\[\]@]+)?/[^/?#\[\]@]+$ # http://iiif.io/api/image/2.1/#uri-encoding-and-decoding, with support for percent encoding
+            pattern: ^https://[^/]+(?:/[^?#\[\]@]+)?/[^/?#\[\]@]+$ # http://iiif.io/api/image/2.1/#uri-encoding-and-decoding, with support for percent encoding
```

Comment:
(previous comments vanished on the outdated diff)

Reply:
to make it strict enough, it may have to mandate no IIIF server prefix hence no `/` after `https://hostname/`

Reply:
We should switch to using proper prefixes rather than Loris's manipulation of the identifier (eg `https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif`).